src/helix-ui/styles/navigation.less

```diff
@@ -15,21 +16,37 @@
   padding-top: 2rem;
   width: 15rem; /* ~240px */
 
-  header {
+  hx-disclosure {
     color: inherit;
-    padding: @vpad @rpad @vpad 0; // left set below
+    display: block;
+    padding: @vpad @rpad @vpad @l1-indent;
     text-transform: uppercase;
+    position: relative;
 
     &:hover {
       background-color: @gray-800;
       color: @gray-0;
       cursor: pointer;
     }
 
+    .icon-transform {
+      transform+_: translateY(-50%);
```

Comment:
The syntax here probably looks strange, but it's a nifty feature of LESS that allows you to merge multiple properties into a single property with multiple values. This case, we're using "Space" merging, meaning we're combining values with a space delimiter (see [less docs](http://lesscss.org/features/#merge-feature-space)).